src/System.Management.Automation/engine/runtime/Binding/Binders.cs

```diff
@@ -6947,15 +6947,7 @@ internal static DynamicMetaObject InvokeDotNetMethod(
                 //  - Log method invocation to AMSI Notifications (can throw PSSecurityException)
                 //  - Invoke method
                 string targetName = methodInfo.ReflectedType?.FullName ?? string.Empty;
-                expr = Expression.Block(
-                    Expression.Call(
-                        CachedReflectionInfo.MemberInvocationLoggingOps_LogMemberInvocation,
-                        Expression.Constant(targetName),
-                        Expression.Constant(name),
-                        Expression.NewArrayInit(
-                            typeof(object),
-                            args.Select(static e => e.Expression.Cast(typeof(object))))),
-                    expr);
+                MaybeAddMemberInvocationLogging(expr, targetName, name, args);
```

Comment:
I may be missing something,  but it looks like the return value from this method call is never used; shouldn't that be `expr =  MaybeAddMemberInvocationLogging(expr, targetName, name, args);`?

Reply:
Good pickup, sorry for missing that. @SeeminglyScience will be submitting a new PR to fix that.

Reply:
That says there are no tests.